go/worker/compute/executor/committee/node.go

```diff
@@ -1848,8 +1850,8 @@ func (n *Node) worker() {
 		case <-txnScheduleTicker.C:
 			// Force scheduling a batch if possible.
 			n.handleScheduleBatch(true)
-		case <-n.scheduleCh.Out():
-			// Regular scheduling of a batch.
```

Comment:
Add a comment back, e.g. "Check any queued transactions and attemt regular scheduling." or similar.